### PR TITLE
Add simple research scripts

### PR DESCRIPTION
**Pull request by michalsidzej**
Adding a script to send L2 transactions. Also modifying batch_decoder to work with devnet. 

I'll work on automating the process a bit more.

TODO: 

- [ ] send deposit
- [ ] withdrat l2 -> l1
- [ ] decode batches automatically

Disclaimer: I am not a golang developer. If you know a better way of doing all of that, please comment.

### CODE REVIEW

File: op-node/cmd/batch_decoder/main.go [hunk 1]
```diff
@@ -186,6 +188,48 @@ func main() {
 				return nil
 			},
 		},
+		{
+			Name:  "reassemble-devnet",
```
Comment (tynes) [inline op-node/cmd/batch_decoder/main.go:192]: We should not add another command that is specific for the devnet, we should add override config to the other command

File: scripts/main.go [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+	l2Url                    = "http://localhost:9545"
+	deployerAddr             = "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"
+	deployerPrivKey          = "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
+	randomAddr               = "0x95222290DD7278Aa3Ddd389Cc1E1d165CC4BAfe5"
+	l1BlockL2ContractAddress = "0x4200000000000000000000000000000000000015"
+)
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "Op devnet scripts"
```
Comment (tynes) [inline scripts/main.go:28]: It is easy to use `cast` for this usecase

**Comment by tynes** [comment]
@michalsidzej Did you intend on actually merging this code? If so, I recommend not adding a new command to the `batch-decoder` and instead adding CLI flags. If the CLI flags are passed, use those values, otherwise read from the registry config.

I also recommend on using `cast` rather than writing scripts in Go for sending transactions. We would prefer to not take that code into the repo

**Comment by michalsidzej** [comment]
> @michalsidzej Did you intend on actually merging this code? If so, I recommend not adding a new command to the `batch-decoder` and instead adding CLI flags. If the CLI flags are passed, use those values, otherwise read from the registry config.
> 
> I also recommend on using `cast` rather than writing scripts in Go for sending transactions. We would prefer to not take that code into the repo

Hi, sorry, I got confused with creating this PR. It's a POC for some devnet automation, wanted to add it to our fork. Sorry for the inconvenience.

**Pull request closed without merging**